### ISSUE TITLE
ReferenceError: global is not defined

### ISSUE THREAD

**Issue by jewe** [tivac/crucible#175]
After installing, configuring config.js and calling `npm start` i got this error in the browser console:
`
Uncaught ReferenceError: global is not defined at browser.js:12
`

**Comment by tivac** [comment]
Could you link the actual line of code from `browser.js` that is failing?

**Comment by jewe** [comment]
in `node_modules/process-es6/browser.js`:


```
if (typeof global.setTimeout === 'function') {
    cachedSetTimeout = setTimeout;
}

```

**Comment by kevinkace** [comment]
Looks like a recent change to `process-es6` might have introduced this issue.

https://github.com/calvinmetcalf/node-process-es6/commit/e68ede3c48498e48391cbda3e8e1433f113e07f7

**Comment by tivac** [comment]
This is fixed in `0.3.4`, which should appear on npm shortly!

I apologize for the delay in getting it fixed, real life got in the way.
